lib/react.js

```diff
@@ -2382,7 +2382,7 @@ module.exports = ReactPropTypesSecret;
 /***/ (function(module, exports, __webpack_require__) {
 
 "use strict";
-/** @license React v16.8.6
+/** @license React v16.8.3
```

Comment:
This should not be getting downgraded. It's possible you'll need to blow away your `package-lock.json` file and re-run `npm install && npm build`.

cc/ @greglittlefield-wf